### PR TITLE
third PR

### PR DESCRIPTION
**Pull request by Nagesh73308**
Review the PR and give the comments on this

### CODE REVIEW

File: e2e/practice/demo.cy.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+describe("test suit(login functionality)",()=>{
+    it.only("test cases(verify the login with valied details)",()=>{
+        cy.visit(Cypress.env('endUrl'))
+        cy.get('[placeholder="Username"]').type('Admin')
+        cy.get('[placeholder="Password"]').type('admin123')
+        cy.contains(' Login ').click()
+        cy.contains('Quick Launch').should('be.visible')
```
Comment (Nagesh73308) [inline e2e/practice/demo.cy.js:7]: please remove this line as well

Comment (Nagesh73308) [reply e2e/practice/demo.cy.js:7]: this is demo purpose

Comment (Nagesh73308) [reply e2e/practice/demo.cy.js:7]: ok

**Review by Nagesh73308 — COMMENTED** [review]
approve

File: e2e/practice/demo.cy.js (hunk 1 above)
Comment (shashikumar79) [inline e2e/practice/demo.cy.js:7]: review

**Review by shashikumar79 — APPROVED** [review]

**Pull request merged**